tests/integration/passthrough-test.js

```diff
@@ -202,3 +202,25 @@ test('it can passthrough other-origin hosts', function(assert) {
   });
 });
 
+
+test('passthrough is not overridable', function(assert) {
```

Comment:
could we rename the test to `passthrough requests follow normal precedence rules`